app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] **Required:** You are required to enable `confirmable` in your authentication strategy. This is to ensure any email account that is provided by users of your application is not a dummy or fake account or an account they don't have access to.